openexp/_canvas/canvas.py

```diff
@@ -634,17 +642,47 @@ def arrow(self, sx, sy, ex, ey, arrow_size=5, color=None, penwidth=None):
 			my_canvas = canvas(exp)
 			w = self.get('width')/2
 			h = self.get('height')/2
-			my_canvas.arrow(0, 0, w, h, arrow_size=10)
+			my_canvas.arrow(0, 0, w, h, arrow_width=40,proportion = 0.8)
 		"""
 
-		self.line(sx, sy, ex, ey, color=color, penwidth=penwidth)
-		a = math.atan2(ey - sy, ex - sx)
-		_sx = ex + arrow_size * math.cos(a + math.radians(135))
-		_sy = ey + arrow_size * math.sin(a + math.radians(135))
-		self.line(_sx, _sy, ex, ey, color=color, penwidth=penwidth)
-		_sx = ex + arrow_size * math.cos(a + math.radians(225))
-		_sy = ey + arrow_size * math.sin(a + math.radians(225))
-		self.line(_sx, _sy, ex, ey, color=color, penwidth=penwidth)
+		# length 
+		d = math.sqrt((ey-sy)**2 + (sx-ex)**2)
+		# direction  
+		angle = math.atan2(ey-sy,ex-sx)
+
+
+		# calculate points between each of the vertices, according to this scheme
+		#    3	
+		#    |\
+		#    | \
+		# 1--2  \
+        # |      \
```

Comment:
Red alert, red alert! *Never* mix space- and tab-based indentation, because Python can't handle it. OpenSesame uses tabs.